app/Transformers/UserTransformer.php

```diff
@@ -53,14 +53,18 @@ public function transform(User $user)
     {
         $profileCustomization = $user->profileCustomization();
 
+        $country = $user->country_acronym === null
```

Comment:
might want to use `presence()` here because `country_acronym` is not nullable (so table has empty strings)

(also might want to do that for the other null checks in this PR too)

Reply:
There's getter in the model for this attrirbute converting empty strings into null.

Reply:
Oh hrm, I think I might have been on the wrong/stale branch when checking this -- disregard!